### ISSUE TITLE
How to edit learning rate?

### ISSUE THREAD

**Issue by EagleYing** [shenweichen/DeepCTR#168]
Please refer to the [FAQ](https://deepctr-doc.readthedocs.io/en/latest/FAQ.html) in doc and search for the [related issues](https://github.com/shenweichen/DeepCTR/issues) before you ask the question.

**Describe the question(问题描述)**
A clear and concise description of what the question is.

**Additional context**

**Operating environment(运行环境):**
 - python version [e.g. 3.6]
 - tensorflow version [e.g. 1.4.0,]
 - deepctr version [e.g. 0.5.2,]

**Comment by shenweichen** [comment]
please refer to the [faq about set learning rate](https://deepctr-doc.readthedocs.io/en/latest/FAQ.html#set-learning-rate-and-use-earlystopping)
